src/components/Hanafiyyah/en/Indeterminate/index.stories.js

```diff
@@ -39,7 +39,8 @@ Default.args = {
   ramadanIcon: 'https://uxwing.com/wp-content/themes/uxwing/download/web-app-development/asterisk-icon.png',
   marriageIcon: 'https://uxwing.com/wp-content/themes/uxwing/download/web-app-development/asterisk-icon.png',
   settingsIcon: 'https://uxwing.com/wp-content/themes/uxwing/download/tools-equipment-construction/setting-icon.png',
-  closeIcon: 'https://uxwing.com/wp-content/themes/uxwing/download/checkmark-cross/close-icon.png'
+  settingsCloseIcon: 'https://uxwing.com/wp-content/themes/uxwing/download/checkmark-cross/close-icon.png',
+  popUpCloseIcon: 'https://uxwing.com/wp-content/themes/uxwing/download/checkmark-cross/close-icon.png'
```

Comment:
Why are we downloading from a slow wordpress site? Can we not download and out on cdn?

Reply:
This is only for storybook so the users can see how to use images url 
![image](https://github.com/user-attachments/assets/42db7a97-1301-435c-b404-0a5ef77fb965)